packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.html

```diff
@@ -0,0 +1,79 @@
+<template>
+  <c-quantic-search-interface engine-id={engineId} skip-first-search>
+    <div class="slds-form-element">
+      <div class="slds-form-element__control">
+        <div class="slds-combobox_container">
+          <div
+            class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click"
+            aria-expanded="true"
+            aria-haspopup="listbox"
+            role="combobox"
+          >
+            <div class="searchbox__container">
+              <div class={searchBoxContainerClass} role="none">
+                <template if:true={withoutSubmitButton}>
+                  <lightning-icon
+                    class="slds-icon slds-input__icon slds-input__icon_left"
+                    icon-name="utility:search"
+                    size="x-small"
+                    title=""
```

Comment:
the title is it required?

Reply:
I guess not, this is a copy from quanticSearchBox.html

Reply:
If it is indeed unnecessary could you remove it from the normal searchbox as well?